tests/test.config.func.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+// eslint-disable-next-line no-unused-vars
+const webpackConfig = (env = {}, argv = {}) => ({
+  entry: {
+    index: path.join(__dirname, 'fixtures', 'index'),
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
```

Comment:
Possibly, but since this a 1-to-1 copy of `test.config.js` I will update this to import that configuration. This file was added to ensure the tests were started as expected when using either a webpack config object or function.